code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -104,14 +104,14 @@
 	. = 1
 
 /datum/reagent/drug/crank/addiction_act_stage3(mob/living/M)
-	M.adjustBruteLoss(5*REM, 0)
+	M.adjustInternalLoss(3*REM, 0)
```

Comment:
b2i

Reply:
unless you'd say I should label crank differently from "some toxins" this is in the cl